content/en/docs/tasks/debug/debug-cluster/audit.md

```diff
@@ -66,7 +66,7 @@ compared against the list of rules in order. The first matching rule sets the
 _audit level_ of the event. The defined audit levels are:
 
 - `None` - don't log events that match this rule.
-- `Metadata` - log request metadata (requesting user, timestamp, resource,
+- `Metadata` - log event metadata (requesting user, timestamp, resource,
```

Comment:
this seems wrong. The metadata is about requests, not about events. The events are the carrier of the metadata, not the source.

Reply:
Similarly, line 68 is not accurate.

Reply:
Maybe this would clarify it for line 69: `log events with request metadata only (....)`

Reply:
Interesting, so if the metadata is about requests then 70/71 are wrong too. Since it uses the same phrasing and that was the root of my confusion.

In terms of all the options except `None` - don't they all provide the same type of metadata, or does the subject of that metadata actually change based on the option? If they provide metadata on the same subject, then the phrasing on all 3 active options should be uniform to be more clear.

Today line 69 reads as the only one referencing the `request metadata` versus `event metadata`.

Reply:
@sttts - I like that suggestion, so keeping with the goal of clearer consistency that would mean the others would read as:

- Metadata - log events with request metadata [...]
- Request - log events with request metadata and body but not response body. [...]
- RequestResponse - log events with request metadata, request body and response body. [...]

Reply:
Of course this might be hairsplitting. Technically this is what we are talking about here:

<img width="582" alt="Bildschirmfoto 2024-10-08 um 14 32 42" src="https://github.com/user-attachments/assets/943ffa8d-ea44-4198-a1d2-dff4cadd5c80">

The field `metav1.TypeMeta`, `Level`, `AuditID` and `Stage` are event metadata.
All the other fields lik `RequestURI`, `Verb`, .... are request metadata, with exception of `RequestObject` and `ResponseObject` which are not logged in `Metadata` mode.

Reply:
Right. The audit module generates audit **events**, based on request/response body and/or metadata. When we say "log events with request metadata only", we are talking about things that are generated. When we say "log request metadata", we are talking about the data source. It is the wording here that causes confusion, IIUC.

Reply:
I've updated this PR per the feedback here. It seemed that the suggestion @sttts provided and I applied to all 3 was going to be acceptable to clarify the wording?